src/features/commonFeatureUtils.js

```diff
@@ -0,0 +1,37 @@
+import { ContextParams } from '../tokenizer';
```

Comment:
I just noticed that the file extensions are missing, but are now required since we changed the build process and the library may be used in module form on the web, where we need the file extensions. I'm fixing it right now.